lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -336,18 +341,19 @@ def revert_json_html_encoding!(string)
       def api_request(method, url, rel_path, params)
         params == {} ? body = '' : body = params.to_json
         revert_json_html_encoding!(body) if defined?(ActiveSupport::JSON::Encoding) && ActiveSupport::JSON::Encoding.escape_html_entities_in_json
-        parse(ssl_request(method, url, body, headers(rel_path, body)))
+        parse(ssl_request(method, url, body, headers(rel_path, body, params)))
```

Comment:
Looks like params is added but never used

Reply:
oh you're right, thanks, I'll update it